### ISSUE TITLE
Icons not displaying in safari

### ISSUE THREAD

**Issue by timmermansjoy** [kausko/Portfolio#10]
When using this template or looking own your own website with safari the icons do not show up

**To Reproduce**
You can use mobile or desktop safari to look at your template / website all buttons are just coloured squares

**Expected behavior**
Icons should display

**Desktop:**
 - OS: [MacOS]
 - Browser [safari]

**Smartphone:**
 - Device: [e.g. iPhone X]
 - OS: [iOS 15]
 - Browser [safari]

**Comment by kausko** [comment]
Hi,

I don't have any Apple devices at the moment, so I was unaware about this issue. I tested this out using browser emulators (like browserstack and lambdatest). TIL, the `width` and `height` attributes need to mentioned explicitly for SVGs to work in Safari. I didn't consider adding these before because the SVG icons provided by `simple-icons` do not have these as well.

Anyway, I fixed this in 262f017137a8688766611f4a7fea20108fdbd633. Output screenshot attached below. Let me know if this worked for you. Thanks @timmermansjoy!

![image](https://user-images.githubusercontent.com/53346282/140655107-5bb61e0d-65ec-47f1-809e-6289dc2bb784.png)
